pkg/kubelet/server/stats/fs_resource_analyzer.go

```diff
@@ -89,7 +89,9 @@ func (s *fsResourceAnalyzer) updateCachedPodVolumeStats() {
 		podUid := pod.GetUID()
 		stats, found := s.getPodVolumeStats(pod)
 		if !found {
-			glog.Warningf("Could not locate volumes for pod %s", format.Pod(pod))
+			if pod.Status.Phase == api.PodRunning {
```

Comment:
We will be collecting stats for all pods that are known to the volume manager regardless of their state.  If a pod is running but not known to volume manager we log a warning.  Filed issue #21416 for logging warnings for pods that are partially running and not known to volume manager.